src/assets/styles/global.scss

```diff
@@ -20,6 +20,12 @@
   --calcite-ui-opacity-disabled: 0.5;
 
   --calcite-panel-width-multiplier: 1;
+  --calcite-alert-dismiss-progress-background: #{rgba($blk-000, 0.8)};
```

Comment:
@bstifle I wonder if in the future we should just add a sequence of transparent blacks to the palette? Seems like most of these are just various amounts of semi-transparent black